scripts/check_lint.sh

```diff
@@ -48,18 +48,30 @@ trap clean EXIT
 for i in $(find $lint_path $exclude -name "go.mod")
 do
   pushd "$(dirname $i)" > /dev/null
-  if [ -z $(go list) ]; then
+  if [ -z $(go list 2> /dev/null) ]; then
+      echo "skipped $(dirname $i)"
       popd  > /dev/null
       continue
   fi
 
-  ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} \
-  -v "$ROOT:/opt/workspace" \
-  -v "$ROOT/.golangci.yml:/opt/workspace/$(dirname $i)/.golangci.yml" \
-  -w "/opt/workspace/$(dirname $i)" \
-  ${lint_image} golangci-lint run
+  if [ -f "$ROOT/$(dirname $i)/.custom_golangci.yml" ]; then
+      # use local lint config
+      ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} \
+      -v "$ROOT:/opt/workspace" \
+      -v "$ROOT/$lint_path/.custom_golangci.yml:/opt/workspace/$(dirname $i)/.custom_golangci.yml" \
+      -w "/opt/workspace/$(dirname $i)" \
+      ${lint_image} golangci-lint run -c "/opt/workspace/$(dirname $i)/.custom_golangci.yml" \
+      --path-prefix "$(dirname $i)/"
```

Comment:
QoL change: golangci running inside a subfolder will now print file:line paths that are clickable in intelliJ, just like when it runs in the main folder